From a60675cc737e169a9724200ed2d805ad249181a1 Mon Sep 17 00:00:00 2001
From: Neal Liu <neal_liu@aspeedtech.com>
Date: Mon, 8 Sep 2025 10:24:46 +0800
Subject: [PATCH 17/32] arch: arm: dts: ast2700-reserved-mem: add
 ipc_bootmcu_share node

Add ipc_bootmcu_share reserved memory node to sync with
linux reserved memory file.

Signed-off-by: Neal Liu <neal_liu@aspeedtech.com>
Change-Id: Id360778cf2cd6474d696f5c0b886ad925b2b4740
(cherry picked from commit e50016f9571d463166dc27a8d0bf369fcbc78d27)
Signed-off-by: Vasanthakumar K <vasanthakumark@ami.com>
---
 arch/riscv/dts/ast2700-ibex-reserved-mem.dtsi | 5 +++++
 1 file changed, 5 insertions(+)

diff --git a/arch/riscv/dts/ast2700-ibex-reserved-mem.dtsi b/arch/riscv/dts/ast2700-ibex-reserved-mem.dtsi
index 5a8b452a33..42159b3c16 100644
--- a/arch/riscv/dts/ast2700-ibex-reserved-mem.dtsi
+++ b/arch/riscv/dts/ast2700-ibex-reserved-mem.dtsi
@@ -73,6 +73,11 @@ ipc_ssp_share: ipc-ssp-share@b1080000 {
 	bootph-all;
 };
 
+ipc_bootmcu_share: ipc-bootmcu-share@b1880000 {
+	reg = <0xb1880000 0x2b0000>;
+	no-map;
+};
+
 uhci0_reserved: uhci0-reserved@b1b30000 {
 	reg = <0xb1b30000 0x40000>;
 	compatible = "shared-dma-pool";
-- 
2.34.1

